### DISCUSSION TITLE
How to create fully custom toast bar

### DISCUSSION THREAD

**Discussion by gamegee** [timolins/react-hot-toast#134] [Q&A] [answered]
Hi

I am working on a large CSS codebase and I want to customize the toast bar.
However it seems not possible to pass `className` directly to the `ToastBar` ?

**Comment by TastyDucks** [comment, marked as answer]
You may add a className or other styles by modifying the ToastOptions in your `useToast` or `Toaster`: https://react-hot-toast.com/docs/toast

If you want to use entirely custom JSX, you can use `toast.custom(<div>Something</div>)`

**Reply by TooniOlaniyan** [reply]
For me, the toast.custom doesnt seem to work
<img width="1440" alt="Screenshot 2024-12-16 at 12 59 08 PM" src="https://github.com/user-attachments/assets/91f7f782-fa5f-49ac-9bce-893c0ff178db" />
